ehrql/tables/raw/core.py

```diff
@@ -37,10 +37,10 @@ class ons_deaths_raw(EventFrame):
 
     In the associated database table a small number of patients have multiple registered deaths.
     This table contains all registered deaths.
-    The `ehrql.tables.ons_deaths` table contains the earliest registered death.
+    The `ehrql.tables.raw.core.ons_deaths` table contains the earliest registered death.
```

Comment:
I think this should be...

```suggestion
    The `ehrql.tables.core.ons_deaths` table contains the earliest registered death.
```

Reply:
Yes, you're right, thanks!